helm/aws-load-balancer-controller/values.yaml

```diff
@@ -389,5 +389,25 @@ ingressClassConfig:
 # enableServiceMutatorWebhook allows you enable the webhook which makes this controller the default for all new services of type LoadBalancer
 enableServiceMutatorWebhook: true
 
+# serviceMutatorWebhook contains configurations specific to the service mutator webhook
+serviceMutatorWebhook:
+  # Specifies whether a service mutator webhook should be created
+  enabled: true
```

Comment:
I think the `serviceMutatorWebhook.enabled` is redundant to  `enableServiceMutatorWebhook`, can you just use `enableServiceMutatorWebhook`? it needs to be true if someone wants to config `serviceMutatorWebhook`